content/logs/aws.md

```diff
@@ -47,10 +47,37 @@ Push your AWS log information to Datadog using a Lambda function bound to an S3
 
 ## Collection
 
-Your lambda function is now ready to send logs to your Datadog platform.
-Setup the relevant triggers for each AWS service you want to monitor.
-
-### S3, Cloudwatch, API Gateway, Kinesis and SNS
+Your lambda function is now ready to send logs to your Datadog platform. Setup the relevant triggers for each AWS service you want to monitor.
+
+**The Lambda function you just created must be in the same region as the S3 bucket you are using as a trigger. If you have logs across multiple regions you must create additional Lambda funcions**
+
+### Automatically set up triggers
+If you are storing logs in many S3 buckets, Datadog can automatically manage triggers for you.
+
+1. Add the required permissions to your Datadog role in the [IAM Console](https://console.aws.amazon.com/iam/home#/roles). You may already have some of these permissions from our other AWS integrations. Information on how these permissions are used can be found in the [permissions](#permissions) section below
+
+{{< highlight json>}}
```

Comment:
this is just a list of things, nothing JSON'y about it. so the commas are being highlighted in a wierd way, but i think it would be worth adding a complete valid IAM policy rather than just the ALLOW clause